Automattic/wp-calypso
# Blog onboarding: Show relevant plans information

Fixes https://github.com/Automattic/dotcom-forge/issues/2489

## Proposed Changes

We want to show relevant features for the flow as shown on Figma and discussed on Slack.

Figma: CZ07QdsC7bmWbLiSO5K9T5-fi-1134%3A53409
Slack: p1684420100443169/1684419959.208059-slack-CKZHG0QCR

## Testing Instructions

* Using a new user OR a user without any site, start the flow with [/setup/start-writing](http://calypso.localhost:3000/setup/start-writing).
* Look at the plans, it should look like Figma.
* Check choosing a free and a paid domain.

## Screenshots
| Before | After |
| --- | --- |
| ![image](https://github.com/Automattic/wp-calypso/assets/402286/a1f581e3-eeb3-457a-a772-05bc387bb89f) ![image](https://github.com/Automattic/wp-calypso/assets/402286/cb93e201-eb2d-4972-a8bf-a89e54175739) | ![image](https://github.com/Automattic/wp-calypso/assets/402286/cf8757d8-ab34-446b-8616-751c3d19d78e) ![image](https://github.com/Automattic/wp-calypso/assets/402286/fde2a265-7125-43e7-957f-696450ad5fdb) |

## Pre-merge Checklist
